layouts/partials/nav.html

```diff
@@ -1,52 +1,83 @@
 <!-- Navbar -->
-    <nav class="navbar navbar-expand-lg bg-primary fixed-top navbar-transparent " color-on-scroll="400">
-        <div class="container">
-            <div class="navbar-translate">
-                <a class="navbar-brand" href="{{ .Site.BaseURL | relLangURL }}" rel="tooltip" title="" data-placement="bottom" target="_blank">
-                    {{.Site.Title}}
-                </a>
+<nav class="navbar navbar-expand-lg bg-primary fixed-top navbar-transparent " color-on-scroll="400">
+  <div class="container">
+    <div class="navbar-translate">
+      <a class="navbar-brand" href="{{ .Site.BaseURL | relLangURL }}" rel="tooltip" title="" data-placement="bottom"
+        target="_blank">
+        {{.Site.Title}}
+      </a>
 
-                <button class="navbar-toggler navbar-toggler" type="button" data-toggle="collapse" data-target="#navigation" aria-controls="navigation-index" aria-expanded="false" aria-label="Toggle navigation">
-                    <span class="navbar-toggler-bar bar1"></span>
-                    <span class="navbar-toggler-bar bar2"></span>
-                    <span class="navbar-toggler-bar bar3"></span>
-                </button>
-            </div>
-            <div class="collapse navbar-collapse justify-content-end" id="navigation" data-nav-image="{{ "img/blurred-image-1.jpg" | relURL }}">
-              <ul class="navbar-nav  ml-auto">
-      {{ range .Site.Menus.main }}
-        {{ if .HasChildren }}
-        <li class="nav-item dropdown">
-          <a href="#" class="nav-link dropdown-toggle"  id="navbarDropdownMenuLink" data-toggle="dropdown">
-            {{ .Pre }} <span>{{ .Name }}</span>
-            <span class="caret"></span>
-          </a>
-          <div class="dropdown-menu dropdown-menu-right" aria-labelledby="navbarDropdownMenuLink">
-            {{ range .Children }}
-            <a class="dropdown-item" href="{{ .URL | relLangURL }}"{{ if $.IsHome }} data-target="{{ .URL }}"{{ end }}>
+      <button class="navbar-toggler navbar-toggler" type="button" data-toggle="collapse" data-target="#navigation"
+        aria-controls="navigation-index" aria-expanded="false" aria-label="Toggle navigation">
+        <span class="navbar-toggler-bar bar1"></span>
+        <span class="navbar-toggler-bar bar2"></span>
+        <span class="navbar-toggler-bar bar3"></span>
+      </button>
+    </div>
+    <div class="collapse navbar-collapse justify-content-end" id="navigation"
+      data-nav-image="{{ "img/blurred-image-1.jpg" | relURL }}">
+      <ul class="navbar-nav my-4 mr-auto">
+        {{ range .Site.Menus.main }}
+          {{ if .HasChildren }}
+          <li class="nav-item dropdown">
```

Comment:
Maybe I'm reading over it. But I don't see here how the current menu item is active.

Reply:
This is pretty much unnecessary if you don't plan to have dropdown items in your menu.